### ISSUE TITLE
Crash Bandicoot 2 n-tranced doesnt save 

### ISSUE THREAD

**Issue by grenade44** [libretro/mgba#48]
I started playing this game and when I choose save from the main menu it doesn't save and I loose my progress, the game saves fine in vba-m. Can someone please look into this and get back to me. Thank you

**Comment by sergiobenrocha2** [comment]
Report this kind of bugs to upstream please

**Comment by sergiobenrocha2** [comment]
I can confirm it, when it's trying to save:

```
RetroArch [libretro DEBUG] :: GBA Memory: Out of bounds ROM Load16: 0x0D000000
```

@endrift

**Comment by sergiobenrocha2** [comment]
```
RetroArch [INFO] :: Saving RAM type #0 to "/home/sergio/Games/RetroArch/saves/mGBA/Crash Bandicoot 2 - N-Tranced (USA).srm".
RetroArch [INFO] :: Saved successfully to "/home/sergio/Games/RetroArch/saves/mGBA/Crash Bandicoot 2 - N-Tranced (USA).srm".
```

**Comment by grenade44** [comment]
@sergiobenrocha2 sorry I'm new to github, what do you mean report to up stream?

**Comment by sergiobenrocha2** [comment]
Upstream is the main repository that we forked it.

**Comment by endrift** [comment]
Fixed upstream in 8cc5339b1e257a1d6fdb9f96f07b474611dea9c8, will be fixed downstream once changes are merged.

**Comment by sergiobenrocha2** [comment]
Merged. Missing AC8J yet, closing
